terraform/eks/e2e/variables.tf

```diff
@@ -21,6 +21,16 @@ variable "nodes" {
   default = 1
 }
 
+variable "ami_type" {
```

Comment:
I am curious do we actually need to use variables for the AMI/Instance type ? Are we testing a variety of AMIs/Instances for EKS ?

Reply:
You're right to question this. I don't think we need this, but I just thought it was worth adding this variable in case we wanted to configure it in the future. I could remove it since AMI testing for EKS doesn't seem necessary for the purpose of these tests. What do you think?